attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
```

Comment:
EOTB is an invis effect, so be aware that it may get dropped by other scripts. I think `release_invisibility` is the function you want to search for - it's used in several other town scripts.